libs/angular-google-charts/src/lib/components/google-chart/google-chart.component.ts

```diff
@@ -210,6 +211,27 @@ export class GoogleChartComponent implements ChartBase, OnChanges, OnInit {
     }
   }
 
+  /**
+   * For listening to events other than the most common ones (available via Output properties).
+   *
+   * Can be called after the chart emits that it's "ready".
+   *
+   * Returns a handle that can be used for `removeEventListener`.
+   */
+  public addEventListener(eventName: string, callback: Function): any {
+    const handle = this.registerChartEvent(this.chart, eventName, callback);
+    this.eventListeners.set(handle, { eventName, callback, handle });
+    return handle;
+  }
+
+  public removeEventListener(handle: any): void {
+    const entry = this.eventListeners.get(handle);
```

Comment:
Do we need this check? If we just try to remove the given `handle` from the chart, everything will work as expected. If the handle is not registered, I guess an error will be shown, but that's somewhat expected in this case.

Reply:
Well we are actually checking for `entry` so that we would use its `handle` property.
Optional chaining could be used instead, but it would call `removeListener` with `undefined` :-)